ui/test/unit/appointments/controllers/manage/list/appointmentsListViewController.spec.js

```diff
@@ -94,6 +94,7 @@ describe('AppointmentsListViewController', function () {
     it('should get appointments for date', function () {
         createController();
         var viewDate = new Date('1970-01-01T11:30:00.000Z');
+		$state.params = {getAppointments: true};
```

Comment:
Hi @krishnanspace,
Indent it properly

Reply:
@krishnanspace I'm confused that @sumanmaity112 had to point out an indentation issue. I would assume that `grunt web` does take care of that sort of things.
* If yes, did you run `grunt web` before pushing your commit?
* If not, then we need to address this (outside of the current ticket of course.)

Reply:
@krishnanspace use the PR first message to explain what you have done. Like
> In this commit:
> * A new boolean flag is introduced stating whether appointment data should be re-fetched or not.
> * Whatever else is relevant.
> * ...

If possible try to put in simple words what your code change is about.